department-of-veterans-affairs/caseflow
# Moved routes into the new folder structure

### Description

This part of the PR stack moves the routes for reader into the new structure. This involved adding lazy loading and some placeholder code for the fallback until we move over the layouts in the next PR.

### Acceptance Criteria

### Testing Plan
1. Shadow user AAABSHIRE and click the Reader tab and then Documnets list on the test users page: http://localhost:3000/test/users
1. Make sure that you can still use Reader the same as before
1. Open a rails console and toggle the feature on: FeatureToggle.enable!(:interface_version_2)
1. Reload the page and ensure you can now use Reader the same as before
